### ISSUE TITLE
Visualization of NIH and NSF grants

### ISSUE THREAD

**Issue by titipata** [titipata/grant_explorer#1]
http://metricsgraphicsjs.org/

**Comment by daniel-acuna** [comment]
cleaner than bokeh also without the annoying logo

**Comment by titipata** [comment]
I agree. This looks much cleaner. We can use it for visualization instead, i.e. convert output summarization to prober json file. Or we can use https://github.com/vega/vega-lite, created by Thai folk.

**Comment by daniel-acuna** [comment]
either one looks fine. it seems the vega-lite is more widely used.
